pkg/apis/certmanager/v1/types.go

```diff
@@ -18,52 +18,65 @@ package v1
 
 // Common annotation keys added to resources.
 const (
-	// Annotation key for DNS subjectAltNames.
+	// AltNamesAnnotationKey is an annotation key for DNS subjectAltNames.
 	AltNamesAnnotationKey = "cert-manager.io/alt-names"
 
-	// Annotation key for IP subjectAltNames.
+	// IPSANAnnotationKey is an annotation key for IP subjectAltNames.
 	IPSANAnnotationKey = "cert-manager.io/ip-sans"
 
 	// Annotation key for URI subjectAltNames.
+	// URISANAnnotationKey is ...
 	URISANAnnotationKey = "cert-manager.io/uri-sans"
 
 	// Annotation key for certificate common name.
+	// CommonNameAnnotationKey is ...
 	CommonNameAnnotationKey = "cert-manager.io/common-name"
 
 	// Duration key for certificate duration.
+	// DurationAnnotationKey is ...
 	DurationAnnotationKey = "cert-manager.io/duration"
 
 	// Annotation key for certificate renewBefore.
+	// RenewBeforeAnnotationKey is ...
 	RenewBeforeAnnotationKey = "cert-manager.io/renew-before"
 
 	// Annotation key for certificate key usages.
+	// UsagesAnnotationKey is ...
 	UsagesAnnotationKey = "cert-manager.io/usages"
 
 	// Annotation key the 'name' of the Issuer resource.
+	// IssuerNameAnnotationKey is ...
 	IssuerNameAnnotationKey = "cert-manager.io/issuer-name"
 
 	// Annotation key for the 'kind' of the Issuer resource.
+	// IssuerKindAnnotationKey is ...
 	IssuerKindAnnotationKey = "cert-manager.io/issuer-kind"
 
 	// Annotation key for the 'group' of the Issuer resource.
+	// IssuerGroupAnnotationKey is ...
 	IssuerGroupAnnotationKey = "cert-manager.io/issuer-group"
 
 	// Annotation key for the name of the certificate that a resource is related to.
+	// CertificateNameKey is ...
 	CertificateNameKey = "cert-manager.io/certificate-name"
 
 	// Annotation key used to denote whether a Secret is named on a Certificate
 	// as a 'next private key' Secret resource.
+	// IsNextPrivateKeySecretLabelKey is ...
 	IsNextPrivateKeySecretLabelKey = "cert-manager.io/next-private-key"
 )
 
 const (
+	// IngressIssuerNameAnnotationKey is ...
 	// issuerNameAnnotation can be used to override the issuer specified on the
```

Comment:
This probably works for this family:
```suggestion
	// IngressIssuerNameAnnotationKey can be used to override the issuer specified on the
```